apps/prairielearn/elements/pl-order-blocks/dag_checker.py

```diff
@@ -248,7 +253,9 @@ def lcs_partial_credit(
                     x for x in submission_no_distractors if x not in subset
                 ]
                 edited_group_belonging = {
-                    key: group_belonging.get(key) for key in edited_submission
+                    key: group_belonging.get(key)
+                    for key in edited_submission
+                    if key is not None
```

Comment:
same thing here, shouldn't need the None check

Reply:
Resolved in [10c9d37](https://github.com/PrairieLearn/PrairieLearn/pull/11249/commits/10c9d37f2093c05684b81aeb37e5f3e32073056f).